<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignatureEditorWidget</class>
 <widget class="QTabWidget" name="SignatureEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="InfosTab">
   <attribute name="title">
    <string>Informations</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="IDLabel">
        <property name="text">
         <string notr="true">IDLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IDEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Details:</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="NameEdit"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="VersionEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Authors:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Contacts:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QTableWidget" name="ContactsTableWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Email</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QPushButton" name="AddContactButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveContactButton">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QComboBox" name="StatusComboBox">
         <item>
          <property name="text">
           <string>experimental</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>beta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>stable</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>License:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QLineEdit" name="LicenseEdit"/>
       </item>
       <item row="11" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="AuthorsTableWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Email</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="AddAuthorButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveAuthorButton">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="DescriptionEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Application(s):</string>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Tags</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TagsEdit">
         <property name="placeholderText">
          <string>domain::water;process::transfer;Tag 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="ParametersTab">
   <attribute name="title">
    <string>Parameters</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="ParametersDataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ExtraFilesGroupBox">
      <property name="title">
       <string>Extra files</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="ExtraFilesDataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="AttributesTab">
   <attribute name="title">
    <string>Attributes</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,1">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Attributes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="AttributesDataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Events</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="EventsDataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="VariablesTab">
   <attribute name="title">
    <string>Variables</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="VariablesDataWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="DynamicsTab">
   <attribute name="title">
    <string>Dynamics</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>Scheduling</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="UndefSchedRadioButton">
           <property name="text">
            <string>Undefined scheduling</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QRadioButton" name="DefaultSchedRadioButton">
           <property name="text">
            <string>Scheduling uses default DeltaT value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="FixedSchedRadioButton">
           <property name="text">
            <string>Scheduling uses a fixed DeltaT value:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FixedSchedValueEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="RangeSchedRadioButton">
           <property name="text">
            <string>Scheduling uses a DeltaT value varying from</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RangeSchedMinValueEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RangeSchedMaxValueEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>Spatial graph</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,1,0,3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Overall description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="SpatialDynDescEdit"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="openfluid::ui::common::SignatureDataEditorWidget" name="SpatialDynDataWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="BuilderextTab">
   <attribute name="title">
    <string>Integration in Builder</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="openfluid::ui::common::BuilderextSetupWidget" name="BextSetupWidget" native="true"/>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="IssuesTab">
   <attribute name="title">
    <string>Issues</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="openfluid::ui::common::WareIssuesManagerWidget" name="IssuesManagerWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>openfluid::ui::common::BuilderextSetupWidget</class>
   <extends>QWidget</extends>
   <header location="global">openfluid/ui/common/BuilderextSetupWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>openfluid::ui::common::SignatureDataEditorWidget</class>
   <extends>QWidget</extends>
   <header location="global">openfluid/ui/common/SignatureDataEditorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>openfluid::ui::common::WareIssuesManagerWidget</class>
   <extends>QWidget</extends>
   <header location="global">openfluid/ui/common/WareIssuesManagerWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
